pkg/runner/cluster_k8s.go

```diff
@@ -160,15 +158,8 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 		}
 	}
 
-	var defaultCPU, defaultMemory resource.Quantity
-	defaultCPU, err := resource.ParseQuantity(cfg.PodResourceCPU)
-	if err != nil {
-		defaultCPU = resource.MustParse("100m")
-	}
-	defaultMemory, err = resource.ParseQuantity(cfg.PodResourceMemory)
-	if err != nil {
-		defaultMemory = resource.MustParse("100Mi")
-	}
+	defaultCPU := resource.MustParse(cfg.TestplanPodCPU)
```

Comment:
@coryschwartz I think we want to panic here if `testplan_pod_cpu` is not set. Hardcoding default values in the codebase is confusing, because if folks misconfigure the `.env.toml` and do not apply these, would find out hours later that their pods have different resource requirements as expected.

This already happened once.